reconcile/typed_queries/fleet_labels.py

```diff
@@ -0,0 +1,14 @@
+from reconcile.gql_definitions.fleet_labeler.fleet_labels import (
+    FleetLabelsV1,
+    query,
+)
+from reconcile.utils import gql
+from reconcile.utils.gql import GqlApi
+
+
+def get_fleet_labels(
+    api: GqlApi | None = None,
+) -> list[FleetLabelsV1]:
+    api = api or gql.get_api()
+    data = query(api.query)
+    return list(data.fleet_labels or [])
```

Comment:
why need to create another `list` instead of directly `data.fleet_labels or []`?

Reply:
Hmm good question. Its a pattern we have in many typed queries - copy/paste. Fixed here now